src/compiler/factory/node_factory/lines_0_1000.rs

```diff
@@ -71,6 +95,26 @@ impl<TBaseNodeFactory: 'static + BaseNodeFactory> NodeFactory<TBaseNodeFactory>
         })
     }
 
+    pub(crate) fn set_converters(
+        &self,
+        node_converters: Box<dyn NodeConverters<TBaseNodeFactory>>,
+    ) {
+        *self.converters.borrow_mut() = Some(node_converters);
+    }
+
+    pub(crate) fn converters(&self) -> Ref<Box<dyn NodeConverters<TBaseNodeFactory>>> {
+        Ref::map(self.converters.borrow(), |option| option.as_ref().unwrap())
+    }
+
+    pub fn create_assignment(
+        &self,
+        base_factory: &TBaseNodeFactory,
+        left: Rc<Node /*Expression*/>,
+        right: Rc<Node /*Expression*/>,
+    ) -> BinaryExpression {
+        self.create_binary_expression(base_factory, left, SyntaxKind::EqualsToken, right)
```

Comment:
There's some kind of currying optimization in the Typescript version but didn't attempt to mimic that here